### PR TITLE
Integrate 11/9 RN Nightly Bulid

### PR DESCRIPTION
**Pull request by chiaramooney**
### Type of Change
- Integration

### What
https://github.com/facebook/react-native/compare/1fc27c4cb...c63882238

- Removal of schedulerDidCloneShadowNode from FabricUIManager
- Removal of fork of AttributedString.cpp.
- Removal of ComponentWithState example.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/10883)

### CODE REVIEW

**Comment by vmoroz** [comment]
Should it be 10/24 instead of 10/10?

**Comment by vmoroz** [comment]
This is the matching hermes-windows version: 0.0.0-2210.24001-35f41a3a

**Review by marlenecota — APPROVED** [review]

File: vnext/Microsoft.ReactNative.Cxx/JSI/NodeApiJsiRuntime.cpp [hunk 1]
```diff
@@ -2061,6 +2061,8 @@ napi_value NapiJsiRuntime::GetNapiValue(const Value &value) const {
     return GetNapiValue(value.getString(*const_cast<NapiJsiRuntime *>(this)));
   } else if (value.isObject()) {
     return GetNapiValue(value.getObject(*const_cast<NapiJsiRuntime *>(this)));
+  } else if (value.isBigInt()) {
+    return GetNapiValue(value.getObject(*const_cast<NapiJsiRuntime *>(this)));
```
Comment (marlenecota) [inline vnext/Microsoft.ReactNative.Cxx/JSI/NodeApiJsiRuntime.cpp:2065]: getBigInt?

**Pull request merged**